ui/app/AppLayouts/Communities/popups/TransferOwnershipPopup.qml

```diff
@@ -1,93 +1,145 @@
+            text: qsTr("To transfer ownership of %1:").arg(root.communityName)
+            font.bold: true
+        }
 
-            input.text: elidedPkey
-            input.edit.readOnly: true
-            input.edit.onActiveFocusChanged: {
-                pKeyInput.input.text =  pKeyInput.input.edit.focus ? root.privateKey : elidedPkey
-            }
-            input.rightComponent: StatusButton {
-                anchors.right: parent.right
-                anchors.rightMargin: Style.current.halfPadding
-                anchors.verticalCenter: parent.verticalCenter
-                borderColor: Theme.palette.primaryColor1
-                size: StatusBaseButton.Size.Tiny
-                text: qsTr("Copy")
-                objectName: "copyCommunityPrivateKeyButton"
-                onClicked: {
-                    text = qsTr("Copied")
-                    Utils.copyToClipboard(root.privateKey)
-                }
-            }
+        CustomText {
+            text: qsTr("1. Send the %1 Owner token (%2) to the new owner’s address").arg(root.communityName).arg(token.name)
+        }
+
+        CustomText {
+            text: qsTr("2. Ask the new owner to setup the control node for %1 on their desktop device").arg(root.communityName)
         }
 
-        StatusBaseText {
+        StatusMenuSeparator {
             Layout.fillWidth: true
+        }
 
-            text: qsTr("You should keep it safe and only share it with people you trust to take ownership of your community")
-            wrapMode: Text.WordWrap
-            font.pixelSize: 13
-            color: Theme.palette.baseColor1
+        CustomText {
+            text: qsTr("I acknowledge that...")
         }
 
-        StatusBaseText {
-            Layout.fillWidth: true
+        StatusCheckBox {
+            id: ackCheckBox
+
+            Layout.topMargin: -Style.current.halfPadding
+            Layout.bottomMargin: Style.current.halfPadding
 
-            text: qsTr("You can also use this key to import your community on another device")
-            wrapMode: Text.WordWrap
-            font.pixelSize: 13
-            color: Theme.palette.baseColor1
+            font.pixelSize: Style.current.primaryTextFontSize
+            text: qsTr("My ownership rights will be removed and transferred to the recipient")
+        }
+    }
+
+    header: StatusDialogHeader {
+        headline.title: qsTr("Transfer ownership of %1").arg(root.communityName)
+        leftComponent: Rectangle {
+            height: 40
+            width: height
+            radius: height/2
+            color:Theme.palette.baseColor2
+
+            Image {
+                id: image
+
+                source: root.communityLogo
+                anchors.fill: parent
+                fillMode: Image.PreserveAspectFit
+                visible: false
+            }
+
+            OpacityMask {
+                anchors.fill: image
+                source: image
+                maskSource: parent
+            }
```

Comment:
I used this in the ExportControlNodePopup.qml as the header:
```qml
    header: StatusDialogHeader {
        headline.title: qsTr("How to move the %1 control node to another device").arg(root.community.name)
        actions.closeButton.onClicked: root.close()
        leftComponent: StatusSmartIdenticon {
            asset.name: root.community.image
            asset.isImage: !!asset.name
        }
    }
```

I believe it's a bit more compact :)

Reply:
Done!